tests.py

```diff
@@ -13,18 +13,16 @@ def test_un():
         r'1	a',
         r'2	\t',
         r'3	\n',
-        r'4	\\',
+        r'4	\\\\',
```

Comment:
Added that case too: https://github.com/solidsnack/tsv/pull/6/commits/0fda23221a1a7d2e88690cbdfc0648b2f7ebafbe